app/boards/arm/bluemicro840/bluemicro840_v1.yaml

```
@@ -13,3 +13,6 @@ supported:
   - ieee802154
   - pwm
   - watchdog
+manufacturer: BlueMicro
+compatible: Pro Micro
```

Using a "friendly name" for this feels really odd. I had expected something like:

```suggestion
compatible: pro_micro
```

or

```suggestion
compatible: promicro
```

---

Totally agree with you on this. I originally had it as you have here, but then I ran into the issue of where do I get the friendly name from? I figured this was good enough for now, but it needs to be addressed.